example-project/sketches/hsl/hsl.js

```diff
@@ -0,0 +1,66 @@
+export default `
```

Comment:
I don't love needing to export it like this. Would prefer a raw .glsl or .isf file, such that you can configure your IDE for syntax highlighting, intellisense, etc.

I tried to import it with a different extension, but the app didn't like that, and I'd rather not need to use a network request if we can avoid that, etc.

Reply:
hmm i'm sure there's a nice solution to this. I vaguely remember there being some way to get syntax highlighting inside the string in VSCode. Maybe some extension. Otherwise it might even be possible with the sketches server to do something clever and have raw files.

Reply:
Or something like this... you use the string but just to do an import of the whole file:
https://github.com/glslify/glslify-import

Reply:
yeah, could use glslify, but being able to grab files as raw would be best, keeps options open for sketches to import anything

Reply:
Been thinking about this and it should be possible with esbuild loaders. By default, `.txt` files are loaded in as strings when imported. Should be able to do the same with `.glsl` somehow...

Reply:
yeah, just a question of ideally not needing to add every extension by hand.

[searching around](https://vitejs.dev/guide/assets#importing-asset-as-string) it sounded like you should be able to just add `?raw` onto the end of the path, but it doesn't seem to work, maybe due to the files being loaded later?

Reply:
The documentation you linked is for vite, but the sketches server is using esbuild. We're using vite for building Hedron, but at runtime we're running esbuild to do the sketches server stuff. What we need is a loader for esbuild, I think I can do this very quick, will make a second PR in a moment!